packages/data-stores/src/plans/types.ts

```diff
@@ -180,6 +191,8 @@ export interface PricedAPIPlanPricing {
 	orig_cost_integer: number;
 
 	currency_code: string;
+
+	cost_overrides?: CostOverrides[];
```

Comment:
Is this applicable to both `/sites/[site]/plans` and `/plans` endpoints? If not, then let's maybe move it to `PricedAPISitePlanPricing` below.